hw/ip/rom_ctrl/data/rom_ctrl_testplan.hjson

```diff
@@ -14,18 +14,36 @@
             Smoke test exercising the main features of rom_ctrl.
 
             **Stimulus**:
-            - Create a random valid ROM image and load into memory model
+            - Create a random valid ROM where expected digest doesn't match with the KMAC digest
+            image and load into memory model
             - Allow the rom check to complete
             - Perform some random memory accesses
 
             **Checks**:
             - Check that all data supplied to kmac is correct
             - Check that the rom checking sequence gives the expected result
             - Check that the memory accesses return expected data
+            - Check that pwrmgr_data_o.good is not asserted. 
             '''
       milestone: V1
       tests: ["rom_ctrl_smoke"]
     }
+    {
+      name: successful_rom_chk
+      desc: '''
+           Test successful completion of rom check
+
+           **Stimulus**:
+           - Create a random valid ROM image and load into memory model.
+           - Ensure that KMAC and expected digest match.
+         
+           **Checks**:
+           - Check that pwrmgr_data_o.good is asserted. 
+           '''
+      milestone: V1
+      tests: ["successful_rom_chk"]
```

Comment:
I don't think you've written a "successful_rom_chk" test yet, right? If so, I think that dvsim will moan at you when you try to run anything for `rom_ctrl`. The correct thing is to leave this list empty (`tests: []`) until the test has been implemented.

Reply:
Rupert, I've written the test and the PR for it is #9330

Reply:
Ok, so you either have to ensure that this PR lands second or (probably easier) leave this empty and either fill it in for this PR when rebasing if the other PR goes in first or fill it in in the other PR if this one goes in first.